gridded_etl_tools/utils/extractor.py

```diff
@@ -311,49 +486,25 @@ def request(self, remote_file_path: str, destination_path: pathlib.Path | None =
             else:
                 destination_path /= file_name
 
-        # Open the remote file, and write it locally
-        response = self.session.get(remote_file_path)
-        response.raise_for_status()
-        with open(destination_path, "wb") as outfile:
-            outfile.write(response.content)
-
-        # If no exceptions were raised, the file was downloaded successfully
-        return True
+        return self.retry_with_backoff(
+            identifier=remote_file_path,
+            remote_file_path=remote_file_path,
+            destination_path=destination_path,
```

Comment:
I'd be in favor of going ahead and changing `destination_path` to `local_file_path`. We can do a very minimal change to managers code to accompany this

Reply:
It would remove an obvious source of confusion and error